### ISSUE TITLE
Minifying the resulting JS file more

### ISSUE THREAD

**Issue by Fusseldieb** [kalkih/mini-graph-card#424]
Hi,
I just saw that in the resulting JS file there are a lot of white spaces, many of them probably unecessary, wasting bandwidth, and every KB saved is a great thing! Doesn't sound like much, but still...

I searched for "at least" double spaces in VSCode and then deleted them to compare the sizes.

Before:
<img src="https://user-images.githubusercontent.com/4715129/90891593-57716100-e392-11ea-9d7d-2aa602893c50.png" width=400>

After:
<img src="https://user-images.githubusercontent.com/4715129/90891611-5dffd880-e392-11ea-94cc-bfca164e7828.png" width=400>

Please note that this might not be 100% accurate, nor have I tested the code for functionality, but most of these spaces could be eliminated without any issue, especially those between html tags, before/after line breaks `\n`) and such.

I'm not just pointing it out. I want to help too. That means if I'm allowed I would clone this repo and include a little [grunt script](https://gruntjs.com/) into it. This script would essentially take the final JS file, erase the spaces following a regex or something and then overwrite it. There are probably other ways, but I was thinking that this would be the straightest route.

EDIT: I just saw that ESLint has some minifying options too, but I'm not familiar with it, I just flown over the related documentation real quick.

**Comment by Fusseldieb** [comment]
> Perhaps https://github.com/cfware/babel-plugin-template-html-minifier ?

Thanks for the suggestion! I tried it, included it into `.babelrc` following [the example for lit-element](https://github.com/cfware/babel-plugin-template-html-minifier#usage), but it didn't minify a thing. Spend coutless hours trying to adjust parameters, but still nothing.

**Comment by kalkih** [comment]
Terser has a [rollup plugin](https://www.npmjs.com/package/rollup-plugin-terser) so if that does the trick I'm all for it 👍

**Comment by Fusseldieb** [comment]
> Terser has a [rollup plugin](https://www.npmjs.com/package/rollup-plugin-terser) so if that does the trick I'm all for it 👍

I used it according to documentation, but for some reason it just removed the comments from the final file, leaving the JS file with all whitespaces in place, making it even bigger. I thought a plugin was interfering with it, so I disabled everything and tried to `rollup-plugin-terser` almost barebone, to no avail.

Testing [Terser](https://github.com/terser/terser) itself in the CLI did in fact minimize the final file and made it 762 Bytes smaller compared to babels "minify". However, the whitespaces after `\n` on the HTML Elements are still there.
